pkgs/tools/misc/tmux/default.nix

```diff
@@ -65,6 +65,14 @@ stdenv.mkDerivation rec {
   postInstall = ''
     mkdir -p $out/share/bash-completion/completions
     cp -v ${bashCompletion}/completions/tmux $out/share/bash-completion/completions/tmux
+  '' + lib.optionalString stdenv.isDarwin ''
+    mkdir -p $terminfo/share/terminfo/74 $out/nix-support
+    cp -v ${ncurses}/share/terminfo/74/tmux $terminfo/share/terminfo/74
+    # macOS ships an old version (5.7) of ncurses which does not include tmux-256color so we need to provide it from our ncurses.
+    # However, due to a bug in ncurses 5.7, we need to first patch the terminfo before we can use it with macOS.
+    # https://gpanders.com/blog/the-definitive-guide-to-using-tmux-256color-on-macos/
+    tic -o $terminfo/share/terminfo -x <(TERMINFO_DIRS=${ncurses}/share/terminfo infocmp -x tmux-256color | sed 's|pairs#0x10000|pairs#32767|')
+    echo "$terminfo" >> $out/nix-support/propagated-user-env-packages
```

Comment:
We can move this to passthru, so we don't need to build tmux to get the terminfo.

Reply:
Updated :+1:

Reply:
Even better, thanks!